host_classifier.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+if [ "$#" -lt 1 ]; then
+echo "please provide at least 1 file"
+fi
+
+var1="0"
```

Comment:
var1, var2, etc are not good and explicit variable names, it would be better to name them after what they actually represent... suggestions would be something like os_windows_count or env_production_count